scripts/install-ci.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -xe
+
+echo "Install protobuf ..."
+wget "https://github.com/protocolbuffers/protobuf/releases/download/v3.14.0/protoc-3.14.0-linux-x86_64.zip"
+unzip protoc-3.14.0-linux-x86_64.zip -d $HOME/.local
```

Comment:
Move the version to a constant?

Reply:
Using the Github Action instead of manually.